### DISCUSSION TITLE
Cannot read property 'graphql' of undefined

### DISCUSSION THREAD

**Discussion by cjnoname** [MichalLytek/type-graphql#915] [Q&A]
The project is hosting on Lambda and built with Webpack 5.

I got this error:

function getPeerDependencyGraphQLRequirement() {
    const ownPackageJson = require("../../package.json");
    return ownPackageJson.peerDependencies.graphql;
}

There is an error that peerDependencies is undefined. Any thoughts?

Thanks

**Comment by Cerchie** [comment]
What does your package.json look like?

**Reply by cjnoname** [reply]
I added
```
  "peerDependencies": {
    "graphql": "15.5.0"
  }
```
to my package.json and it works. Not sure whether it's the right approach

**Reply by MichalLytek** [reply]
@cjnoname When you build the app with webpack, the `package.json` of TypeGraphQL from `node_modules` is missing thus throwing the error on strict peer dependency check. You can try to provide a path to the file in webpack assets or just use your workaround.
